docs/guide/tutorial-yii-integration.md

```diff
@@ -62,12 +62,18 @@ In the worst case when the library requires explicitly including every class fil
 to include the classes on demand:
 
 * Identify which classes the library contains.
-* List the classes and the corresponding file paths in `Yii::$classMap` in the [entry script](structure-entry-scripts.md)
-  of the application. For example,
-```php
-Yii::$classMap['Class1'] = 'path/to/Class1.php';
-Yii::$classMap['Class2'] = 'path/to/Class2.php';
-```
+* List the classes and the corresponding file paths in the application `composer.json`:
+  ```json
+  "autoload": {
+      "psr-4": {
+          "app\\": ""
+      },
+      "classmap": [
+          "path/to/Class1.php",
+          "path/to/Class2.php",
+      ]
```

Comment:
`classmap` supports also directories, so sometimes this could be simplified to 

```json
    "classmap": [
        "path/to/library/",
    ]
```

Composer will automatically find all classes in directory and include them into classmap.